test/axes/baseAxisTests.ts

```diff
@@ -266,6 +266,16 @@ describe("BaseAxis", () => {
         assert.strictEqual(axis.annotationTierCount(annotationTierCount), axis, "setting annotation tier count returns calling axis");
         assert.deepEqual(axis.annotationTierCount(), annotationTierCount, "can set the annotation tier count");
       });
+
+      it("throws an error when annotation tier count is not valid", () => {
+        let axis = new Plottable.Axis(new Plottable.Scale<{}, number>(), "bottom");
+        let annotationTierCount = [-1, -100];
+        annotationTierCount.forEach(function(tierCount) {
+          let currentTierCount = axis.annotationTierCount();
+          assert.throw(() => axis.annotationTierCount(tierCount), `annotationTierCount ${tierCount} is not valid, should be positive or 0`);
```

Comment:
We should keep consistent on which of `assert.throw` / `assert.throws` we use.  I see `assert.throws` over and over if I remember correctly.  We should consistently use one of the two and then ban the other.